node/modules/storageminer.go

```diff
@@ -1023,3 +1030,53 @@ func ExtractEnabledMinerSubsystems(cfg config.MinerSubsystemConfig) (res api.Min
 	}
 	return res
 }
+
+func MakeDealStaging(lr repo.LockedRepo, subdir string) error {
```

Comment:
Let's make this method private, and have it take the absolute path to the new directory and the old directories, instead of taking a relative path and the repo.

Reply:
Sounds good. Addressed.